### PR TITLE
blue square logo added

### PR DESCRIPTION
**Pull request by digiacomo4**
What do you think of this logo?

### CODE REVIEW

**Review by digiacomo4 — COMMENTED** [review]
New Logo proposed

**Review by digiacomo4 — COMMENTED** [review]
Logo changed

**Comment by AlvaroCubi** [comment]
@digiacomo4 Hello Matteo, great job with the logo. Before merging could you please add __pycache__/ and *.db to the .gitignore file? This is to avoid including those extra files in the commit, we dont need to track them.

**Review by digiacomo4 — COMMENTED** [review]
.gitignore file modified as suggested

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 docs/build/
+__pycashe__/
```
Comment (AlvaroCubi) [inline .gitignore:2]: this looks like a typo, it should say pycache not pycashe

File: __pycache__/cad.cpython-311.pyc
Comment (AlvaroCubi) [inline __pycache__/cad.cpython-311.pyc:1]: should not be tracked

File: __pycache__/free.cpython-311.pyc
Comment (AlvaroCubi) [inline __pycache__/free.cpython-311.pyc:1]: should not be tracked

File: docs/source/_static/Thumbs.db
Comment (AlvaroCubi) [inline docs/source/_static/Thumbs.db:1]: should not be tracked

**Review by digiacomo4 — COMMENTED** [review]
*.pyc and *.db untracked

**Comment by AlvaroCubi** [comment]
Very cool @digiacomo4 !

**Pull request merged**